client/my-sites/patterns/home/style.scss

```
@@ -0,0 +1,58 @@
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.is-section-patterns .layout__content {
+	padding: 0;
+}
+
+.section-patterns-info {
+	overflow: scroll;
```

```suggestion
	overflow: auto;
```
Same thing again about `overflow: scroll` vs `overflow: auto`